projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3140,5 +3140,265 @@ public string FatherEthnicityLiteral
+        /// </example>
+        [Property("Attendant Name", Property.Types.String, "Birth Certification", "Name of attendant.", true, VR.IGURL.Practitioner, true, 6)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner)", "name")]
+        public string AttendantName
+        {
+            get
+            {
+                if (Attendant != null && Attendant.Name != null)
+                {
+                    return Attendant.Name.First().Text;
+                }
+                return null;
+            }
+            set
+            {
+                if (Attendant == null)
+                {
+                    CreatePractitioner();
+                }
+                HumanName name = Attendant.Name.FirstOrDefault();
+                if (name != null && !String.IsNullOrEmpty(value))
+                {
+                    name.Text = value;
+                }
+                else if (!String.IsNullOrEmpty(value))
+                {
+                    name = new HumanName();
+                    name.Use = HumanName.NameUse.Official;
+                    name.Text = value;
+                    Attendant.Name.Add(name);
+                }
+            }
+        }
+
+        /// <summary>Attendants NPI</summary>
+        /// <value>the attendants npi</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleBirthRecord.AttendantNPI = "1234567890";</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Attendants NPI: {ExampleBirthRecord.AttendantNPI}");</para>
+        /// </example>
+        [Property("Attendants NPI", Property.Types.String, "Birth Certification", "Attendant's NPI.", true, VR.IGURL.Practitioner, true, 13)]
+        [FHIRPath("Bundle.entry.resource.where($this is Practitioner).identifier.where(system='http://hl7.org/fhir/sid/us-npi')", "value")]
+        public string AttendantNPI
+        {
+            get
+            {
+                return Attendant?.Identifier?.Find(id => id.System == "http://hl7.org/fhir/sid/us-npi")?.Value;
+            }
+            set
+            {
+                if (Attendant == null)
+                {
+                    CreatePractitioner();
+                }
+                if (Attendant.Identifier.Count > 0)
+                {
+                    Attendant.Identifier.Clear();
+                }
+                Attendant.Identifier.RemoveAll(iden => iden.System == CodeSystems.US_NPI_HL7);
+                if (String.IsNullOrWhiteSpace(value))
+                {
+                    return;
+                }
+                Identifier npi = new Identifier();
+                npi.Type = new CodeableConcept(CodeSystems.HL7_identifier_type, "NPI", "National Provider Identifier", null);
+                npi.System = CodeSystems.US_NPI_HL7;
+                npi.Value = value.Replace("-", string.Empty).Replace(" ", string.Empty);
```

Comment:
is this formatting requirement documented anywhere in the IG? Does IJE usually include dashes and so they need to be removed?

Reply:
I think you're right, I can remove that formatting (I pulled it from the formatting for SSN, but I don't think NPI needs it)